azure/dt-azure-log-forwarder-terraform-deployment/main.tf

```diff
@@ -0,0 +1,32 @@
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "rg_to_deploy_logfw" {
+  name     = "rg-${local.deployment_name}"
+  location = local.location
+}
+
+module "application" {
+  source                      = "./modules/infrastructure/function"
+  deployment_name             = local.deployment_name
+  location                    = local.location
+  dynatrace_url               = local.dynatrace_url
+  dynatrace_access_key        = local.dynatrace_access_key
+  event_hub_name              = local.event_hub_name
+  event_hub_connection_string = local.event_hub_connection_string
+  require_valid_certificate   = local.require_valid_certificate
+  version_number              = local.version_number
+  resource_group              = azurerm_resource_group.rg_to_deploy_logfw.name
+
+  storage_account_name       = module.storage-blob.azurerm_storage_account_name
+  storage_account_access_key = module.storage-blob.azurerm_storage_account_key
+}
+
+module "storage-blob" {
```

Comment:
I would recommend using underscores `_` instead of dash `-`.
so instead of
```hcl
module "storage-blob" {
}
```
use
```hcl
module "storage_blob" {
}
```

Reply:
Any particular reason? 
I think - for folder names is compliant with project naming convention.

Reply:
![image](https://github.com/user-attachments/assets/4c90e334-df13-4e05-af89-9da46a526a6f)

Reply:
It's a naming best practice convention. I don't know if that is official, but it seems pretty standard: https://www.terraform-best-practices.com/naming#general-conventions. Some linters mark this as a warning.

Reply:
This one more official: https://developer.hashicorp.com/terraform/language/style#resource-naming